lms/djangoapps/edxnotes/tests.py

```diff
@@ -369,8 +393,13 @@ def test_search_correct_data(self, mock_get):
         Tests the result if correct data is received.
         """
         mock_get.return_value.content = json.dumps({
-            "total": 2,
-            "rows": [
+            "count": 2,
```

Comment:
I'm seeing this dict being used in many places, can we define it to a single place and use that instead.